command_config.json

```diff
@@ -0,0 +1,52 @@
+{
+    "type": {
+        "rand_issue": {
+            "git": {
+                "repository": "config['repo']",
+                "closed": "False",
+                "pull_request": "False",
+                "include_labels": "[]",
+                "exclude_labels": "['status: stale']",
+                "sort": "'created'",
+                "ascending_order": "False",
+            },
+            "discord": {
+                "game_name": "'Finding Issues'",
+                "title": "f'{number_of_issues} random picks out of {len(issues)}:'",
+                "error_channel": "default_error_handler",
+            },
+        },
+        "review_PR": {
+            "git": {
+                "repository": "config['repo']",
+                "closed": "False",
+                "pull_request": "True",
+                "include_labels": "['needs: code review']",
+                "exclude_labels": "['status: stale']",
+                "sort": "'updated'",
+                "ascending_order": "True",
+            },
+            "discord": {
+                "game_name": "'On The Cue'",
+                "title": "f'Oldest {number_of_issues} PR' + (f' for @{author}:' if author else ' in review queue:')",
```

Comment:
That's bad. Don't use f-strings here. Use normal strings, and the `.format()` on a string. Supplying extra variables in format doesn't affect the output.

```python
title = config['discord']['title'].format(number_of_issues=number_of_issues,
    author=author)
```

Reply:
Much needed. :+1: